types/react-text-mask/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"dt-header": true,
```

Comment:
You can just make this file `{ "extends": "dtslint/dt.json" }` now (those rules are all enabled by default)

Reply:
:+1: @andy-ms  done